src/pytorch_lightning/README.md

```diff
@@ -126,28 +126,28 @@ pip install pytorch-lightning['extra']
 conda install pytorch-lightning -c conda-forge
 ```
 
-#### Install stable 1.6.x
+#### Install stable 1.7.x
 
-the actual status of 1.6 \[stable\] is following:
+The actual status of 1.7 \[stable\] is the following:
 
-![CI basic testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20basic%20testing/badge.svg?branch=release%2F1.5.x&event=push)
-![CI complete testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20complete%20testing/badge.svg?branch=release%2F1.5.x&event=push)
-![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=release%2F1.5.x&event=push)
-![TPU tests](https://github.com/PyTorchLightning/pytorch-lightning/workflows/TPU%20tests/badge.svg?branch=release%2F1.5.x&event=push)
-![Docs check](https://github.com/PyTorchLightning/pytorch-lightning/workflows/Docs%20check/badge.svg?branch=release%2F1.5.x&event=push)
+[![CI base testing](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-base.yml/badge.svg?branch=%27release/pytorch%27&event=push)](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-base.yml)
+[![CI complete testing](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-full-1.8.yml/badge.svg?branch=%27release/pytorch%27&event=push)](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-full-1.8.yml)
+[![PyTorch & Conda](https://github.com/Lightning-AI/lightning/actions/workflows/ci-pytorch_test-conda.yml/badge.svg?branch=%27release/pytorch%27&event=push)](https://github.com/Lightning-AI/lightning/actions/workflows/ci-pytorch_test-conda.yml)
+[![TPU tests](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-tpu.yml/badge.svg?branch=%27release/pytorch%27&event=push)](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-tpu.yml)
```

Comment:
not 100% sure but I think this needs to be updated
https://github.com/Lightning-AI/lightning/blob/0cbfe08a41a5f517a27fd432d4770feb7bef37af/.actions/setup_tools.py#L73

here the badge refers to the version.

Reply:
because right now it doesn't show anything
https://github.com/Lightning-AI/lightning/tree/docs/link-updates/src/pytorch_lightning#install-stable-17x

Reply:
And what should I do? I have no clue what all this does. I would appreciate direct help @rohitgr7 @Borda or drop this for another PR.

Reply:
let's do a follow-up.
I already mention, we can ignore some if it get's out of scope :)

Reply:
I'll make sure to update all of the badges in #13917!